libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -206,7 +206,16 @@ const AP_Param::GroupInfo AP_Mount::var_info[] = {
 
     // 23 formerly _K_RATE
 
-    // 24 is AVAILABLE
+    // @Param: _RETRAC_ALT
+    // @DisplayName: Altitude at which to retract the mount
```

Comment:
This is likely a bit long for a display name, we could probably go with _Altitude to retract mount_. Then we can expand in the description if needed.